### DISCUSSION TITLE
Extension running order.

### DISCUSSION THREAD

**Discussion by Symbiomatrix** [vladmandic/sdnext#649] [Q&A] [answered]
Hello vlad, I asked this on auto's board a few weeks back and received no answer, so figured I'd try here.
How does your version handle extension call order, for example if multiple extensions override the same process() method? Is there some hint as to which should run earlier / later, beyond the rather arbitrary extension's name? It seems to be causing some issues over at regional prompter / wildcards and dynamic prompt.

**Comment by vladmandic** [comment, marked as answer]
scripts are ordered first by sections:
1. `/scripts`
2. `/extensions-builtin`
3. `/extensions`

and within each of those sections, its alphabetical.

i just wrote a framework to support custom ordering, but need to design ui for it.
as a test, you can create `.priority` in extension folder with a single line which is a two-digit number.
default value is 50 (lower number is higher priority)

**Reply by Symbiomatrix** [reply]
Fantastic, thanks. I'll leave a note on it.
Edit: Do you also plan to allow multiple entry points for the same extension, or per function?

**Comment by TheSeventhCode** [comment]
What is the status on this? @vladmandic 
Does this still work with the .priority or is there a task on working on such a UI?
Currently running into issues when using adetailer and roop at the same time (adetailer runs first, then roop, preferably the other way around).

**Reply by vladmandic** [reply]
it still works based on `.priority` file in extension folder.
there is no ui for that as demand has been really low.
